### ISSUE TITLE
utterance commentary function does not work?!

### ISSUE THREAD

**Issue by soerenthomsen** [OceanGlidersCommunity/Oxygen_SOP#234]
Hi @jbusecke,

during a workshop we just realised that the commentary function below our beloved SOPs is gone?
Any idea what happened?

Cheers

**Comment by callumrollo** [comment]
I've done a bit of digging. Looks like the culprit is the div name of sections. UItterances identifies sections presumably to create a seperate set of comments for each one.

Inspect element shows this block is throwing an error

```javascript
    sections = document.querySelectorAll("div.section");
    if (sections !== null) {
        section = sections[sections.length-1];
        section.appendChild(script);
    }
```

It's looking for divs with class "section"
 In the Nitrate SOP, where comments are working, we do see these divs, e.g.

`<div class="section" id="questions">`

meanwhile, over in the Oxygen SOP, the same object is called a section, rather than being a div with class section 

`<section id="questions">`

I'm sure there's an easy way to solve this and I'm having a go, but my html-fu is rudimentary at best.

**Comment by callumrollo** [comment]
Even changing the JS to search for section, it still throws an error later on. Can't figure out why jupyter-books changed to using section rather than div.section :/

**Comment by soerenthomsen** [comment]
Thanks @callumrollo for digging! 

It is strange but maybe helps to find the error that indeed the commentary function works for the nitrate SOP but not for the oxygen and salinity.  And even more confusing not for the Depth Average Current SOP as we have not changed anything there for a long time. 

@jbusecke and myself pushed quite some PR's lately to the Oxygen SOP to create the pdf (just for Oxygen SOP so far) and improve the citation style (both for oxygen and salinity).

@jbusecke any idea what might have happened? 

I cannot say since when exactly the commentary function is off unfortunately. Realised it yesterday during the workshop!

**Comment by soerenthomsen** [comment]
Here @callumrollo fixed it quickly for the salinity SOP ahead of the workshop starting now. 
https://github.com/OceanGlidersCommunity/Salinity_SOP/pull/139

**Comment by jbusecke** [comment]
> Even changing the JS to search for section, it still throws an error later on. Can't figure out why jupyter-books changed to using section rather than div.section :/

This would be an excellent question to ask over there! Great job digging out the reason for the failure!

**Comment by callumrollo** [comment]
I've filed an Issue with jupyter-book. It's late in the day here, but I'll try making a minimum reproducible example tomorrow

**Comment by jbusecke** [comment]
Amazing. Thanks for the work. Glad there was an intermediate fix!

**Comment by soerenthomsen** [comment]
Thanks @callumrollo and @jbusecke for your support on this!

**Comment by soerenthomsen** [comment]
So we keep this open for now until we find a long term fix?

**Comment by callumrollo** [comment]
Yeah, let's see what comes out of the jupyter-books Issue. Pinning on a previous version isn't ideal and could create problems down the line. It's the best fix we've got for now though.

https://github.com/executablebooks/jupyter-book/issues/1762
